package.json

```diff
@@ -0,0 +1,37 @@
+{
+  "name": "js-weather-app",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "dev": "webpack --mode development",
+    "build": "webpack",
```

Comment:
Let us kindly fix our `build` script as we did to the `dev` script. Kindly add `--mode production` to the build script as we see in the dev script :muscle: